framework/source/class/qx/event/message/Bus.js

```diff
@@ -269,42 +306,45 @@ qx.Class.define("qx.event.message.Bus",
     },
 
     /**
-     * dispatch message and call subscribed functions
+     * Dispatch message, which calls subscribers
      *
-     * @param msg {qx.event.message.Message} message which is being dispatched
-     * @return {Boolean} <code>true</code> if the message was dispatched,
-     *    <code>false</code> otherwise.
+     * @param message {qx.event.message.Message} Message which is being dispatched
+     * @return {Boolean} If the message could be dispatched, i.e. if subscribers
+     * exist which have received the message
      */
-    dispatch : function(msg)
+    dispatch : function(message)
     {
       var sub = this.getSubscriptions();
-      var msgName = msg.getName();
+      var msgName = message.getName();
       var dispatched = false;
 
-      for (var key in sub)
+      for (var topic in sub)
       {
-        var pos = key.indexOf("*");
-
-        if (pos > -1)
+        var l = topic.length;
+        if (topic[l-1] === "*")
         {
-          // use of wildcard
-          if (pos === 0 || key.substr(0, pos) === msgName.substr(0, pos))
+          // use of wildcard, only allowed as "*" or at the end of the topic
+          if (l === 1 || topic.substr(0, l-2) === msgName.substr(0, l-2))
```

Comment:
`l` looks very much like `1`. Suggest renaming the variable to `len`.

Reply:
Done.